src/components/AccountLoginForm.jsx

```diff
@@ -75,7 +76,7 @@ export const AccountLoginForm = props => {
 
   // Submit button state
   const submitEnabled =
-    (dirty && isValid) ||
+    (dirty && isValid && allRequiredFieldsAreFilled) ||
```

Comment:
Why `allRequiredFieldsAreFilled` is not a condition to set `isValid` to `true` ? In my mind, a valid form is a form having all its required fields filled. What to you think ?